extensions/http-provisioner/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ dependencies {
     api(project(":spi:transfer-spi"))
     api(project(":spi:web-spi"))
     implementation(project(":extensions:api:auth-spi"))
+    implementation(project(":extensions:api:auth-tokenbased"))
```

Comment:
I'm not sure this is a good idea, in this way we'll have the http-provisioned forced to use the token based auth.
I would provide a deafult pass through authentication service with the `@Provider(isDefault = true)` annotation on the http webhook extension

Reply:
I will create a new issue for this, so we get this PR closed, then you can take a look next week

Reply:
#1358